### PR TITLE
reinvent eks announcements

### PR DESCRIPTION
**Pull request by spara**
kickoff post to reinvent EKS week

### CODE REVIEW

File: content/blog/reinvent-2020-eks-announcements/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: "re:invent 2020 EKS Feature Releases"
```
Comment (ericrudder) [inline content/blog/reinvent-2020-eks-announcements/index.md:2]:
```suggestion
title: "re:Invent 2020 EKS Feature Releases"
```

File: content/blog/reinvent-2020-eks-announcements/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+date: 2020-12-07
+meta_desc: "A recap of the re:invent 2020 EKS announcements and their impacts"
```
Comment (ericrudder) [inline content/blog/reinvent-2020-eks-announcements/index.md:4]:
```suggestion
meta_desc: "A recap of the re:Invent 2020 EKS announcements and their impacts"
```

File: content/blog/reinvent-2020-eks-announcements/index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+authors:
+    - sophia-parafina
+tags:
+    - kubernetes
+    - EKS
+    - re:invent 2020
+
+---
+
+Amazon announced several Elastic Kubernetes Service feature releases and updates during the first week of AWS re:invent 2020. If we look at all the announcements as a whole, we can see the Kubernetes ecosystem maturing to make deployments and management easier for organizations. Let's take a look at how they can benefit your use of EKS.
```
Comment (ericrudder) [inline content/blog/reinvent-2020-eks-announcements/index.md:15]:
```suggestion
Amazon announced several Elastic Kubernetes Service feature releases and updates during the first week of AWS re:Invent 2020. If we look at all the announcements as a whole, we can see the Kubernetes ecosystem maturing to make deployments and management easier for organizations. Let's take a look at how they can benefit your use of EKS.
```

Comment (ericrudder) [inline content/blog/reinvent-2020-eks-announcements/index.md:15]:
```suggestion
Amazon announced several Elastic Kubernetes Service feature releases and updates during the first week of AWS re:Invent 2020. If we look at all the announcements as a whole, we can see the Kubernetes ecosystem maturing to make deployments and management easier for organizations. Let's take a look at how they can benefit your use of EKS.
```

File: content/blog/reinvent-2020-eks-announcements/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    - sophia-parafina
+tags:
+    - kubernetes
+    - EKS
+    - re:Invent 2020
+
+---
+
+Amazon announced several Elastic Kubernetes Service feature releases and updates during the first week of AWS re:Invent 2020. If we look at all the announcements as a whole, we can see the Kubernetes ecosystem maturing to make deployments and management easier for organizations. Let's take a look at how they can benefit your use of EKS.
+
```
Comment (zchase) [inline content/blog/reinvent-2020-eks-announcements/index.md:16]:
```suggestion

{{% notes %}}
Want some hands-on training with EKS? Join our workshop on Thursday, December 10th where you will create a production-ready EKS cluster, install add-ons to prepare your cluster for developers, and finally ship an application to your cluster. [Register Now]({{< relref "/resources/building-a-kubernetes-platform-in-amazon-eks" >}})
{{% /notes %}}
```

☝️ CTA for the workshop. cc// @isaac-pulumi @lindsaymarolich

**Review by zchase — APPROVED** [review]
Approved via @lindsaymarolich

**Pull request merged**